state_machine/plugs/advanced/TransitionFunctions.py

```diff
@@ -1,3 +1,11 @@
 def announcer(source, destination, cubesat):
     print(f'We are transitioning from {source} -> {destination}!!!')
     print(f'We also have access to the cubesat object: {cubesat}')
+
+
+def low_power_on(source, destination, cubesat):
+    print('We should be turning off most of the cubesat\'s power hungry devices')
```

Comment:
When we were testing in the lab, the code was actually turning things off, right? Can we get that implemented here before merging this?

Reply:
So, we're not sure actually. In theory it should have been turning off the IMU sensors but in reality we have no idea if this is true. What is currently the case is that the radio & sd card stop being used in the low power state (those were responsible for the large spikes anyways). Let me know what you think we should do, I don't really want to merge untested code into main.